itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenariosAcrossInstances.java

```diff
@@ -376,4 +386,63 @@ public void testIncrementalMetadataReplication() throws Throwable {
             "custom.value\t "
         });
   }
+
+  @Test
+  public void testBootStrapDumpOfWarehouse() throws Throwable {
+    String randomOne = RandomStringUtils.random(10, true, false);
+    String randomTwo = RandomStringUtils.random(10, true, false);
+    String dbOne = primaryDbName + randomOne;
+    String dbTwo = primaryDbName + randomTwo;
+    WarehouseInstance.Tuple tuple = primary
+        .run("use " + primaryDbName)
+        .run("create table t1 (i int, j int)")
+        .run("create database " + dbOne)
+        .run("use " + dbOne)
+        .run("create table t1 (i int, j int) partitioned by (load_date date) "
+            + "clustered by(i) into 2 buckets stored as orc tblproperties ('transactional'='true') ")
+        .run("create database " + dbTwo)
+        .run("use " + dbTwo)
+        .run("create table t1 (i int, j int)")
+        .dump("`*`", null, Arrays.asList("'hive.repl.dump.metadata.only'='true'",
+            "'hive.repl.dump.include.acid.tables'='true'"));
+
+    /*
+      Due to the limitation that we can only have one instance of Persistence Manager Factory in a JVM
+      we are not able to create multiple embedded derby instances for two different MetaStore instances.
+    */
+
+    primary.run("drop database " + primaryDbName + " cascade");
+    primary.run("drop database " + dbOne + " cascade");
+    primary.run("drop database " + dbTwo + " cascade");
+
+    /*
+       End of additional steps
+    */
+
+    replica.run("show databases")
+        .verifyFailure(new String[] { primaryDbName, dbOne, dbTwo })
+        .load("", tuple.dumpLocation)
```

Comment:
REPL LOAD may need to throw error if the dump have multiple Dbs and we try to load with DBName specified as input.

Reply:
well determining the same would be additional work and will only happen during execution time anyways. hence letting the execution fail itself when its used with dbname and dump contains multiple db's

Reply:
Shall document this behaviour.